### PR TITLE
⚙️ fix: downgrade vm op api to fix ci

### PR DESCRIPTION
**Pull request by XudongLiuHarold**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,10 @@
 module k8s.io/cloud-provider-vsphere
 
-go 1.22.4
+go 1.22.0
 
-replace github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels => github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels v1.8.7-0.20240625173706-a93aa9f0bda0
+toolchain go1.22.3
+
+replace github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels => github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels v1.8.7-0.20240530162623-88d9b71d9741
```
Comment (DanielXiao) [inline go.mod:7]: This should also be v1.8.6

Comment (XudongLiuHarold) [reply go.mod:7]: Is it okay to have any random version, or do you have to set it to align with the vm-operator version?

Comment (DanielXiao) [reply go.mod]: It does not report an error but better to align with vm-operator/api version

**Pull request merged**